### ISSUE TITLE
Examples for the libaesnd?

### ISSUE THREAD

**Issue by Wohlstand** [devkitPro/wii-examples#8]
#### Overview:
Recently I found that in addition to the `asnd` the `aesnd` library exists which exists for a while. Why not to provide the example on how to use it?

### Why would this feature be useful?
Examples should provide a way to showcase the way how the `aesnd` can be used and allows to compare it with the well-known `asnd` thing.

**Comment by dakotath** [comment]
I might attempt to write an example for AESND, but I'm not too sure what would be an interesting demo. Could you maybe provide an idea of a showcase for AESND?

**Comment by Wohlstand** [comment]
You may just repeat existing demos, or make one demo that plays one or multiple streams, plays SFX, etc.

**Comment by heartbytee** [comment]
> You may just repeat existing demos, or make one demo that plays one or multiple streams, plays SFX, etc.

Yeah, it would be great to have separate repeated examples for aesnd using the existing audio players from the asnd examples, like mp3player, oggplayer, or modplay.
